src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,15 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService x = new UserService();
+        User y = x.findByEmail(email);
```

Comment:
```suggestion
        User user = userService.findByEmail(email);
```
Do not use abbreviations;)